packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -1438,17 +1458,48 @@ export class IntervalCollection<TInterval extends ISerializableInterval>
             this.client.rebasePosition(end, sequenceNumber, localSeq);
 
         const intervalId = properties?.[reservedIntervalIdKey];
+        const localInterval = this.localCollection.getIntervalById(intervalId);
+
         const rebased: ISerializedInterval = {
             start: startRebased,
             end: endRebased,
             intervalType,
             sequenceNumber: this.client?.getCurrentSeq() ?? 0,
             properties,
         };
+
         if (opName === "change" && (this.hasPendingChangeStart(intervalId) || this.hasPendingChangeEnd(intervalId))) {
             this.removePendingChange(serializedInterval);
             this.addPendingChange(intervalId, rebased);
         }
+
+        if (!localInterval) {
+            return rebased;
+        }
+
+        // TODO: message
+        assert(localInterval instanceof SequenceInterval, "");
+
+        // if the interval slid off the string, rebase the op to be a nop and
```

Comment:
```suggestion
        // if the interval slid off the string, rebase the op to be a noop and
```
(we use the two-O spelling elsewhere)